src/it/java/teammates/it/storage/sqlapi/AccountRequestsDbIT.java

```diff
@@ -88,4 +89,45 @@ public void testUpdateAccountRequest() throws Exception {
                 accountRequest.getEmail(), accountRequest.getInstitute());
         verifyEquals(accountRequest, actual);
     }
+
+    @Test
+    public void testSqlInjectionInEmailField() throws Exception {
+        ______TS("SQL Injection test in email field");
+
+        // Attempt to use SQL commands in email field
+        String email = "name'; DROP TABLE account_requests; --@gmail.com";
```

Comment:
Maybe try an input that does not fail the regex?

Try
```java
String email = "blabla'/**/OR/**/1=1/**/OR/**/'blabla'='@gmail.com";
```

(inspired by @kevin9foong)

From [RegExr](https://regexr.com/):

![image](https://github.com/TEAMMATES/teammates/assets/65202977/f14ce0a5-ee90-472c-97ae-f7251b0f171a)

From [some random online PostgreSQL online editor I found](https://onecompiler.com/postgresql/425fpp43g):

![image](https://github.com/TEAMMATES/teammates/assets/65202977/b0b035c9-6172-49a1-85dc-5eadce983b13)


Or adjust it accordingly if it still fails unless there's really no way. The original regex does not work for me on RegExr even after un-escaping the backslashes because there is a "+" after a "*" quantifier, so I'm not sure if I got that right, or if maybe our regex is bugged, or if maybe I do not understand regex on Java.

Reply:
Oh right yup, forgot to change that. Thanks!